src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -587,11 +596,14 @@ object SbtBundle extends AutoPlugin {
   private def formatServiceName(serviceName: String): String =
     s"""        service-name  = "$serviceName""""
 
-  private def formatEndpoints(endpoints: Map[String, Endpoint]): String = {
+  private def formatEndpoints(bundleConfVersion: BundleConfVersions.Value, endpoints: Map[String, Endpoint]): String = {
     val formatted =
       for {
```

Comment:
Because there is a lot of code within the `yield` block - I guess it's a style thing?